### ISSUE TITLE
Erroneous Thermal Runaway Errors During Probing

### ISSUE THREAD

**Issue by SeldomRight** [alexqzd/Marlin#2]
Hello. I believe there may be an issue with the firmware causing the nozzle and bed Temps to drop during probing; in my instance with a bltouch. Reproducible even after PID tunings, new thermistor, and new heating element. This only occurs when the heaters are on and during probe. Printer maintains temperatures at all other times. Please see attached screenshots. The first is a normal G28 Z probe and the other from during a full G29 4x4 probe. Thank you!
![Screenshot_20211017-162739_Chrome](https://user-images.githubusercontent.com/93007764/138507954-e9e65957-0e95-4154-9eae-ff710679be8b.jpg)

![Screenshot_20211018-125309_Chrome](https://user-images.githubusercontent.com/93007764/138507830-54696383-a85e-4122-817c-830a85db2c93.jpg)

**Comment by Dpapesca** [comment]
no-one has answered this issue? I'm having the same problem. either the auto bed leveling will just drop the bl touch mid air and retract multiple times and then tell me its finished or it will give me thermal runaway warnings and halt. does anyone have any ideas?
